### DISCUSSION TITLE
save ip camera streaming thorugh rtsp-simple-server

### DISCUSSION THREAD

**Discussion by sajidehsan** [bluenviron/mediamtx#1820] [Questions]
i replace this content with everything written in path section of .yml configuration file
paths:
  mypath:
    runOnReady: ffmpeg -rtsp_transport tcp -i rtsp://admin:dpl123321@192.168.66.101:8554/live/ch00_0 -vcodec copy -acodec copy -f segment -reset_timestamps 1 -segment_time 30 -segment_format mkv -segment_atclocktime 1 -strftime 1 /tmp/recording/%Y%m%dT%H%M%S.ts
    runOnReadyRestart: yes
 This configuration file present in "/etc" directory. i run mediamtx by this command
 mediamtx /etc/mediamtx.yml
 but its not saving streaming . please tell me how i can do this

**Comment by dbuscaino** [comment]
Hi i suppose `/live/ch00_0` is your path name so in order to record it you should write something like that:
```
paths: 
live/ch00_0:
runOnReady: ffmpeg -rtsp_transport tcp -i rtsp://admin:dpl123321@192.168.66.101:8554/live/ch00_0 -vcodec copy -acodec copy -f segment -reset_timestamps 1 -segment_time 30 -segment_format mkv -segment_atclocktime 1 -strftime 1 /tmp/recording/%Y%m%dT%H%M%S.ts
runOnReadyRestart: yes
```

Remember to create the directory in which you save the videos if it doesn't exists.
If you use a service to run the server you should probably restart it.
Also pay attention to the directory in which you save videos because in /tmp i think they will be deleted during shutdown

**Reply by sajidehsan** [reply]
thank you for replying.
after changing this 
paths:
  live/ch00_0:
    runOnReady: ffmpeg -i rtsp://localhost:$RTSP_PORT/$RTSP_PATH -c copy -f segment -strftime 1 -segment_time 900 -segment_format mpegts /root/recording/"%Y-%m-%d_%H-%M-%S_%s.ts"
    runOnReadyRestart: yes
    
  it still not recording the streaming . i also changed the directory

**Reply by dbuscaino** [reply]
```
paths:
live/ch00_0:
runOnReady: ffmpeg -i rtsp://localhost:$RTSP_PORT/$RTSP_PATH -c copy -f segment -strftime 1 -segment_time 900 -segment_format mpegts /root/recording/%Y-%m-%d_%H-%M-%S_%s.ts
runOnReadyRestart: yes
```
Pay attention because you added quotation marks on the file name (maybe this is why it doesn't work on your pc).
I tried the above command on linux and it works fine

[a comment by an automated account is not shown]
